mula/scheduler/schedulers/schedulers/boefje.py

```diff
@@ -788,3 +788,50 @@ def calculate_deadline(self, schedule: models.Schedule) -> models.Schedule:
             schedule.deadline_at = datetime.now(timezone.utc) + timedelta(minutes=plugin.interval)
 
         return super().calculate_deadline(schedule)
+
+    def pop_item_from_queue(
+        self, limit: int | None = None, filters: storage.filters.FilterRequest | None = None
+    ) -> list[models.Task]:
+        items = self.queue.pop(limit, filters)
+        if items is None or len(items) == 0:
```

Comment:
Since you're checking for both `None` and emptiness:
```suggestion
        if not items:
```